client/web/web.go

```diff
@@ -618,6 +625,29 @@ func (s *Server) serveAPI(w http.ResponseWriter, r *http.Request) {
 		newHandler[noBodyData](s, w, r, alwaysAllowed).
 			handle(s.proxyRequestToLocalAPI)
 		return
+	case path == "/serve/items":
+		peerAllowed := func(data serveItem, peer peerCapabilities) bool {
```

Comment:
nit: I think using `peer` as the var here is pretty confusing, since that pretty much always means a peer node.  I didn't notice it before, but `apiHandler.getPeer()` is similar.  I would expect that to have something to do with peer nodes, rather than peer capabilities.  I guess it does make `peer.canEdit(featureName)` read very naturally, but outside of that it's a little weird.